From bf7fc44598432394a1b8d91b927c2fd8e54f65bc Mon Sep 17 00:00:00 2001
From: Dan Winship <danwinship@redhat.com>
Date: Wed, 21 Aug 2024 13:17:04 -0400
Subject: [PATCH] UPSTREAM: <carry>: Fix default skips in
 test-kubernetes-e2e.sh

It was still skipping [Skipped:Network/OpenShiftSDN] even though the
tests use ovn-kubernetes now.

The only test that is currently marked [Skipped:Network/OVNKubernetes]
was also marked [Skipped:Network/OpenShiftSDN], which is why it still
worked, but this meant that we were skipping other tests that
_shouldn't_ have been skipped on ovn-k. (Of course, any problems that
resulted from this would be caught in other e2es later, so this wasn't
actually permanently masking any problems.)
---
 openshift-hack/test-kubernetes-e2e.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openshift-hack/test-kubernetes-e2e.sh b/openshift-hack/test-kubernetes-e2e.sh
index 9292de265a0..ea005aee55a 100755
--- a/openshift-hack/test-kubernetes-e2e.sh
+++ b/openshift-hack/test-kubernetes-e2e.sh
@@ -25,7 +25,7 @@ esac
 # openshift-tests will check the cluster's network configuration and
 # automatically skip any incompatible tests. We have to do that manually
 # here.
-NETWORK_SKIPS="\[Skipped:Network/OpenShiftSDN\]|\[Feature:Networking-IPv6\]|\[Feature:IPv6DualStack.*\]|\[Feature:SCTPConnectivity\]"
+NETWORK_SKIPS="\[Skipped:Network/OVNKubernetes\]|\[Feature:Networking-IPv6\]|\[Feature:IPv6DualStack.*\]|\[Feature:SCTPConnectivity\]"
 
 # Support serial and parallel test suites
 TEST_SUITE="${TEST_SUITE:-parallel}"
-- 
2.47.0

